src/hotspot/share/nmt/virtualMemoryTracker.hpp

```diff
@@ -289,13 +313,14 @@ typedef LinkedListIterator<CommittedMemoryRegion> CommittedRegionIterator;
 int compare_committed_region(const CommittedMemoryRegion&, const CommittedMemoryRegion&);
 class ReservedMemoryRegion : public VirtualMemoryRegion {
  private:
-  SortedLinkedList<CommittedMemoryRegion, compare_committed_region>
-    _committed_regions;
-
   NativeCallStack  _stack;
-  MemTag           _mem_tag;
+  MemTag         _mem_tag;
 
  public:
+  bool is_valid() { return base() != (address)1 && size() != 1;}
+  ReservedMemoryRegion() :
+    VirtualMemoryRegion((address)1, 1), _stack(NativeCallStack::empty_stack()), _mem_tag(mtNone) { }
```

Comment:
Style: Space between `is_valid` and constructor, fix initializer list as in `CommittedMemoryRegion`.

Reply:
Done.